<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditShapeDialog</class>
 <widget class="QDialog" name="EditShapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="widthEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="heightLabel">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="heightEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="rotationLabel">
     <property name="text">
      <string>Rotation Angle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="rotationEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="perimeterLabel">
       <property name="text">
        <string>Perimeter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="perimeterOutput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="areaLabel">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="areaOutput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="submitDialogBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>submitDialogBox</sender>
   <signal>accepted()</signal>
   <receiver>EditShapeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitDialogBox</sender>
   <signal>rejected()</signal>
   <receiver>EditShapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
